lzy/allocator/src/main/java/ai/lzy/allocator/alloc/MountDynamicDiskAction.java

```diff
@@ -64,6 +73,8 @@ protected void notifyFinished() {
             } catch (Exception e) {
                 log().error("{} Failed to start unmount dynamic disk action", logPrefix(), e);
             }
+        } else {
+            log().info("{} Mount disk finished", logPrefix());
```

Comment:
add disk_id (or some info about this disk), plz

Reply:
disk_id is already in prefix, that's why I got rid of any redundant data